lib/view_model/deserialize_context.rb

```diff
@@ -1,15 +1,29 @@
 class ViewModel
   class DeserializeContext
-    attr_accessor :updated_associations, :parent_ref
+    attr_accessor :updated_associations, :parent_context, :parent_viewmodel
 
-    def initialize(parent_ref: nil)
-      self.parent_ref = parent_ref
+    def initialize(*)
```

Comment:
Yes, because subclasses pass up arguments that get ignore. Better argument is perhaps that they _shouldn't_.